src/Symfony/Component/Console/Tests/Input/ArgvInputTest.php

```diff
@@ -183,7 +183,17 @@ public function provideInvalidInput()
             array(
                 array('cli.php', 'foo', 'bar'),
                 new InputDefinition(),
-                'Too many arguments.',
+                'No argument expected, got "foo".',
+            ),
+            array(
+                array('cli.php', 'foo', 'bar'),
+                new InputDefinition(array(new InputArgument('number'))),
+                'Too many arguments, expected arguments "number".',
+            ),
+            array(
+                array('cli.php', 'foo', 'bar', 'zzz'),
+                new InputDefinition(array(new InputArgument('number'), new InputArgument('county'))),
+                'Too many arguments, expected arguments "number, county".',
```

Comment:
Reading this error message, I'm wondering if the comma should be removed: `Too many arguments, expected arguments "number" "county".`